src/command-dispatcher.ts

```diff
@@ -0,0 +1,82 @@
+import {  } from "./server";
+
+const {createUsersTable, createUser, createLikedLanguagesTable, 
```

Comment:
These imports become much more readable (and easy on the eyes) if you do something like:
```
const {
  a,
  b,
  c,
  d
} = require('./whatever');
```
It also makes the 80 character limit issue no longer a thing (unless you name functions like in Java, which you shouldn't)

Reply:
I can't seem to find the empty import line in the develop branch code. I don't understand why is it appearing for you.

Ok, understood :)
I will change that